magento1-module/app/code/local/Divante/VueStorefrontBridge/controllers/RewriteController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require_once Mage::getModuleDir('controllers', 'Divante_VueStorefrontBridge') . DS . 'AbstractController.php';
+class Divante_VueStorefrontBridge_RewriteController extends Divante_VueStorefrontBridge_AbstractController
+{
+    public function targetAction()
+    {
+        if (!$this->_checkHttpMethod('GET')) {
+            return $this->_result(405, 'Method not allowed');
+        }
+
+        $requestPath = $this->getRequest()->getParam('request_path');
+        if ($requestPath) {
+            $reader = Mage::getSingleton('core/resource')->getConnection('core_read');
+
+            $select = $reader->select()
+                             ->from('core_url_rewrite', ['target_path'])
+                             ->where('request_path = ?', $requestPath)
+                             ->where('store_id IN (?)', [Mage_Core_Model_App::ADMIN_STORE_ID, (int)Mage::app()->getStore()->getId()])
```

Comment:
I personally avoid using the globally set store id, had some issues in the past with having the wrong store.
might we require it to be set in the URL as param?

Reply:
Here i think we should have both, the `Mage::app()->getStore()->getId()` as a fallback when no storeId is set in the request? 

Like for example:
```
$storeId = $this->getRequest()->getParam('storeId');
if (empty($storeId)) {
    $storeId = (int)Mage::app()->getStore()->getId();
}
```

What do you think about that? @sandermangel